RELEASING.md

```diff
@@ -91,11 +91,11 @@ In order to keep this relevant & accurate, we improve this doc during the
 release process. It's the best time to pause, observe how it all fits together,
 and improve it. We want small, constant improvements which compound. Therefore:
 
-- [ ] Open this doc in your code editor. As you go through these steps, toggle
+- [x] Open this doc in your code editor. As you go through these steps, toggle
```

Comment:
Are these meant to be checked here?

Reply:
I check these pff as I go through the steps - it forces me to slow down and make sure that I didn't skip any steps. I almost always leave them in the other state: one release they are `[ ]`, the next release they switch back to `[x]`, and so on.

Just so that this stops being confusing for reviewers, I will set them back to `[ ]` and add one last action item to remind the person going through this to set the checkboxes back to `[ ]` when done.

Thank you both!